packages/wallet/src/redux/__tests__/test-scenarios.ts

```diff
@@ -228,3 +233,24 @@ export const ledgerCommitments = {
     turnNum: 6,
   },
 };
+
+// Direct funding states
+const initialFundingState = (ourIndex: PlayerIndex, fundingRequestChannelId: string) => {
+  const total = twoThree.reduce(addHex);
+  const safeToDepositLevel = ourIndex === PlayerIndex.A ? '0x0' : twoThree[1];
+  const requiredDeposit = twoThree[ourIndex];
+
+  const action = actions.internal.directFundingRequested(
+    fundingRequestChannelId,
+    safeToDepositLevel,
+    total,
+    requiredDeposit,
+    ourIndex,
+  );
+  return directFundingStates.initialDirectFundingState(action, EMPTY_SHARED_DATA).protocolState;
```

Comment:
It seems strange to me that you would use wallet code to construct a test scenario. Why not construct it explicitly?

Reply:
Mostly convenience over having to manually recreate and maintain a direct funding state.